lib/consyncful/version.rb

```diff
@@ -1,5 +1,5 @@
 # frozen_string_literal: true
 
 module Consyncful
-  VERSION = '0.6.2'
+  VERSION = '0.6.3'
```

Comment:
Given this is a new feature it should be a minor version. 0.7.0

Reply:
All done @mrGrazy